### PR TITLE
Fixes #36358 - Translation improvements

### PR DESCRIPTION
**Pull request by stejskalleos**

### CODE REVIEW

File: locale/action_names.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Autogenerated!
-_("Action with sub plans")
-_("Import Puppet classes")
-_("Import facts")
 _("Remote action:")
+_("»Action with sub plans«")
+_("»Import Puppet classes«")
+_("»Import facts«")
```
Comment (adamruzicka) [inline locale/action_names.rb:5]: Latest tasks should be adding a newline at the end

Comment (stejskalleos) [reply locale/action_names.rb:5]: I have latest `develop`

Comment (adamruzicka) [reply locale/action_names.rb:5]: tasks don't have develop?

Comment (ekohl) [inline locale/action_names.rb:5]: Why do these get `»` and `«`? I think it's the "mark for translation" setting that adds this, but is it leading to double translations?

**Review by ekohl — CHANGES_REQUESTED** [review]

File: locale/action_names.rb (hunk 1 above)
Comment (stejskalleos) [reply locale/action_names.rb:5]: Interesting, it wasn't there before I updated `foreman-tasks` plugin. I will fix it

**Comment by stejskalleos** [comment]
@ekohl @adamruzicka Updated, now it should be ok. I hope.

**Review by adamruzicka — COMMENTED** [review]
The changes look sane, but I'm not sure what's up with the tests

**Comment by stejskalleos** [comment]
Okay, so changes in the translation somehow cause Ruby tests to fail. I don't touch `*.rb` files at all, so I'm kinda baffled about what and why is happening ... In other PRs everything is fine

**Comment by nofaralfasi** [comment]
> Okay, so changes in the translation somehow cause Ruby tests to fail. I don't touch `*.rb` files at all, so I'm kinda baffled about what and why is happening ... In other PRs everything is fine

Rerunning the tests fixed that :)

**Comment by stejskalleos** [comment]
> Rerunning the tests fixed that :)

Huh, I did rerun the tests, didn't help. I observed the same behavior on my local setup (latest `master`), where when I was running tests on a fresh database dynflow failed with an error. The second run was okay though.

**Comment by stejskalleos** [comment]
@adamruzicka It's :green_apple:

**Comment by adamruzicka** [comment]
Thank you @stejskalleos !

**Pull request merged**